cmd/registry/cmd/compute/conformance_test.go

```diff
@@ -117,11 +64,11 @@ func TestConformance(t *testing.T) {
 			conformancePath: filepath.Join("testdata", "styleguide.yaml"),
 			getPattern:      "projects/conformance-test/locations/global/apis/petstore/versions/1.0.0/specs/openapi.yaml/artifacts/conformance-openapitest",
 			wantProto: func() *rpc.ConformanceReport {
-				conformance := initReport(t)
-				conformance.Name = "projects/conformance-test/locations/global/apis/petstore/versions/1.0.0/specs/openapi.yaml/artifacts/conformance-openapitest"
-				conformance.StyleguideName = "openapitest"
+				report := conformance.InitReport(t)
```

Comment:
Done, got rid of test-helpers.go

```
Why did we choose to use a fixed length GuidelineReportGroups list with enum-specific indices instead of having one field for each status group? The latter seems like a much less complicated way to represent the same structure.
```
I agree with your point. I am not completely aware of what factors were into play while designing the original protos. Can we have a separate discussion on this, it feels out of scope for this PR.

Reply:
Right, I agree that shouldn't be changing in this PR.